ntoskrnl/include/internal/i386/mm.h

```diff
@@ -163,7 +163,7 @@ C_ASSERT(PD_COUNT == 1);
 
 #define MI_SYSTEM_PTE_BASE (PVOID)MiAddressToPte(NULL)
 
-/* Base addreses for page directories */
+/* Base addresses for page directories */
```

Comment:
Unrelated change outside translations.

Reply:
It's not translation, it's previous spell checking corrections.